README.rst

```diff
@@ -24,22 +24,22 @@ Features
 
 * **Managing upgrades**: Provides an advanced view for upgrading
   third-party Plone packages using Generic Setup.
-  It allows to upgrade multiple packages at once with an easy to use user
+  It enables upgrading multiple packages at once with an easy to use user
   interface.
   By resolving the dependency graph it is able to optimize the upgrade
   step order so that the upgrade is hassle free.
 
 * **Writing upgrades**: The package provides a base upgrade class with
-  various helpers for tasks often done in upgrades.
+  various helpers for common upgrade tasks.
 
 * **Upgrade directories with less ZCML**: By registering a directory
-  as upgrade-directory, no more ZCML is needed for each upgrade step.
+  as upgrade-directory, no additional ZCML is needed for each upgrade step.
   By using a timestamp as version number we have less (merge-) conflicts
   and less error potential.
 
-* **Import profile upgrade steps**: Some times an upgrade step does simply
-  import an upgrade step generic setup profile, especially made for this
-  upgrade step. A new ZCML directive makes this much simpler.
+* **Import profile upgrade steps**: Sometimes an upgrade step consists
+  solely of importing a purpose-made generic setup profile.  A new
```

Comment:
there seem to be two whitespaces in `profile.  A new`

Reply:
✅